src/Makefile

```diff
@@ -25,6 +25,16 @@ CFLAGS+=	-D_POSIX_C_SOURCE=200809L
 
 CFLAGS+=	-I core/ -I index/ -I utils/
 
+# 
+# Allow some room to build on DEBUG on
+#
+ifeq ($(DEBUG),1)
+CFLAGS+=	-Wno-unused-function 
```

Comment:
These shouldn't be here. You can use `__unused` (or an old school way of `(void)var;`) to mark unused variables or parameters in debug mode.